README.md

```diff
@@ -1 +1,870 @@
-# gmelius-package
\ No newline at end of file
+<table>
+    <thead>
+    <tr>
+        <th>Title</th>
+        <th>Last Updated</th>
+        <th>Summary</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>Gmelius package</td>
+        <td>September 1, 2023</td>
+        <td>Detailed description of the API of the Gmelius package.</td>
+    </tr>
+    </tbody>
+</table>
+---
+title: Gmelius endpoint
```

Comment:
The header table is regenerated with the new format, the old table can be deleted.